ot/unbalanced.py

```diff
@@ -15,10 +17,9 @@
 
 from .backend import get_backend
 from .utils import list_to_array
-# from .utils import unif, dist
 
 
-def sinkhorn_unbalanced(a, b, M, reg, reg_m, method='sinkhorn', numItermax=1000,
+def sinkhorn_unbalanced(a, b, M, reg, reg_m=None, method='sinkhorn', numItermax=1000,
```

Comment:
sinkhorn unbalanced should always requite reg_m it is in thje name. abut indeed do the balanc"d one for reg_m=[inf,inf]